src/index.ts

```diff
@@ -61,4 +72,15 @@ export function activate() {
 const preProcess = (/** @type {string} */source) =>
     source
         .replace(/\</g, '&lt;')
-        .replace(/\>/g, '&gt;');
\ No newline at end of file
+        .replace(/\>/g, '&gt;');
+
+function injectMermaidTheme(md) {
+    const render = md.renderer.render;
+    md.renderer.render = function() {
+        return `<span id="${configSection}"
+                    darkModeTheme="${vscode.workspace.getConfiguration(configSection).get('darkModeTheme')}"
```

Comment:
Make sure to sanitize the config values here. We should only allow the `enum` values listed in the `package.json`

Reply:
Done!